packages/playground/ssr-vue/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "example-external-component": "file:example-external-component",
+    "vant": "^3.3.2",
```

Comment:
Is it possible to remove `vant` as a dependency here? And create a minimal component inlined in the repo instead

Reply:
Has updated @patak-dev 😊